### DISCUSSION TITLE
Espressif ESP32S3: Can't find hal/usb_hal.h

### DISCUSSION THREAD

**Discussion by gotfredsen** [adafruit/tinyuf2#393] [Q&A]
When I try to make a board this way: https://github.com/adafruit/tinyuf2/tree/master/ports/espressif#build
`make BOARD=adafruit_feather_esp32s3 all`
I get the below error:
```c
/Users/bg/Documents/domino4/TinyUF2/tinyuf2/ports/espressif/boards/boards.c:30:10: fatal error: hal/usb_hal.h: No such file or directory
   30 | #include "hal/usb_hal.h"
      |          ^~~~~~~~~~~~~~~
compilation terminated.
```
I have tried various IDF 5.0.6, 5.1.4., 5.2.2 (stable), and 5.3 (beta 2), but more or less the same problem. I feel I am missing a crucial installation... btw: `python tools/get_deps.py espressif` doesn't install anything.

**Comment by hathach** [comment]
the current code use IDF v5.1.1 as mentioned in the same doc.
